var/spack/repos/builtin/packages/fftw/package.py

```diff
@@ -63,14 +63,15 @@ class Fftw(Package):
     depends_on('mpi', when='+mpi')
     depends_on('automake', type='build', when='+pfft_patches')
     depends_on('autoconf', type='build', when='+pfft_patches')
+    depends_on('libtool', type='build', when='+pfft_patches')
 
-    def install(self, spec, prefix):
+    def configure(self, spec, prefix):
         # Base options
         options = [
             '--prefix={0}'.format(prefix),
             '--enable-shared',
             '--enable-threads'
-        ]
+        ] + self.configure_args()
```

Comment:
`self.configure_args()` will only contain `--prefix`, right? That means it will be duplicated.